digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/assetUpdater/LaneUpdater.scala

```diff
@@ -83,115 +106,148 @@ object LaneUpdater {
     }
   }
 
-  def updateChangeSet(changeSet: ChangeSet) : Unit = {
+  def updateChangeSet(changeSet: ChangeSet, roadLinkChanges: Seq[RoadLinkChange] = Seq(), generateReports: Boolean = false): Seq[ChangedAsset] = {
```

Comment:
generateReports: Boolean = false
Hyym missä tilanteessa tarvitaan tätä, ei tää nyt pahaa tee.

Reply:
jos true, niin muodostaa noita ChangedAsset tallennuksien yhteydessä

Reply:
Boolean arvo sen takia, että fillTopology tallennukset tallennetaan tällä samalla metodilla, ja niiltä ei löydy kaikkia tietoja ChangedAsset muodostamiseen, joten ei haluta näissä tapauksissa koittaa raportteja muodostaa

Reply:
eikös metodin tekeminen näille kahdelle tilanteelle sitten ratko tän tongelman. Se olis musta siistimpi ratkaisu kuin ylläpitää kahta eri moodia.

Reply:
Muutenkin oon vahvasti sitä mieltä että näitä updater metodeita ei saisi kutsua muualta koodipohjasta. Tekee turhaan tiukan sidoksen

Reply:
Joo voisihan nuo erottaa kahdeksi eri metodiksi, teen näin